customerClient/src/main/java/org/lilachshop/customerclient/App.java

```diff
@@ -1,39 +1,177 @@
 package org.lilachshop.customerclient;
```

Comment:
I assume you have validated the correctness of your files, I'm just looking for conflicts.